plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProjectTest.java

```diff
@@ -12,6 +22,27 @@
 
   @Mock private IAdaptable adaptable;
 
+  private SubMonitor monitor = SubMonitor.convert(new NullProgressMonitor());
+  private IProject project;
+  
+  @Before
+  public void setUp() throws CoreException {
+    IWorkspace workspace = ResourcesPlugin.getWorkspace();
+    project = workspace.getRoot().getProject("foobar");
+  }
+  
+  @After
+  public void cleanUp() throws CoreException {
+    project.delete(true, monitor);
+  }
+  
+  @Test
+  public void testConstructor() throws InvocationTargetException, CoreException {
+    AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+    config.setProject(project);
+    new CreateAppEngineStandardWtpProject(config, adaptable);
```

Comment:
I did add a test like that. It passed in Eclipse. However it failed with a NullPointerException deep inside the Eclipse classes when run from maven. Haven't figured that out yet.